src/app/shared/components/message/message.component.html

```diff
@@ -0,0 +1,3 @@
+<div appMessageAnimation *ngIf="msg | async" class="clearfix {{ (msg | async ).type ? (msg | async ).type + ' letIn' : '' }}" id="message">
```

Comment:
again try to subscribe to observable only once and reuse its value

Reply:
Done